auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtension.java

```diff
@@ -82,9 +81,25 @@ public Boolean nullable() {
 
   @Override
   public boolean applicable(Context context) {
+    TypeMirror autoValueClass = context.autoValueClass().asType();
+    // Disallow manual implementation of the CREATOR instance
+    if (!needsCreator(context)) {
+      context.processingEnvironment().getMessager().printMessage(Diagnostic.Kind.ERROR,
```

Comment:
To get the variable element to pass to this exception, I'm thinking of changing the `needsCreator` method to return the variable element itself (or null). Your suggestion below was to change the name of `needsCreator` to `hasCreator`, but it might not make as much sense if it doesn't return a boolean anymore. Would `findCreator` work?